crates/subspace-runtime-primitives/src/weights/balance_transfer_check_extension.rs

```diff
@@ -35,15 +35,13 @@ impl<T: frame_system::Config> crate::extension::WeightInfo for WeightInfo<T> {
 	/// Storage: `RuntimeConfigs::EnableBalanceTransfers` (r:1 w:0)
 	/// Proof: `RuntimeConfigs::EnableBalanceTransfers` (`max_values`: Some(1), `max_size`: Some(1), added: 496, mode: `MaxEncodedLen`)
 	/// The range of component `c` is `[0, 5000]`.
-	fn balance_transfer_check_multiple(c: u32, ) -> Weight {
+	fn balance_transfer_check_multiple(_c: u32, ) -> Weight {
 		// Proof Size summary in bytes:
 		//  Measured:  `0`
 		//  Estimated: `1486`
-		// Minimum execution time: 1_000_000 picoseconds.
-		Weight::from_parts(1_139_820, 0)
+		// Minimum execution time: 1_300_000 picoseconds.
+		Weight::from_parts(1_444_223, 0)
 			.saturating_add(Weight::from_parts(0, 1486))
-			// Standard Error: 12
-			.saturating_add(Weight::from_parts(189, 0).saturating_mul(c.into()))
```

Comment:
This looks like there was something else CPU-heavy running at the same time as this benchmark. Did you check using `top` or similar?

The symptoms of high CPU usage are:
- Increased minimum execution time
- Unrealistically low incremental execution times (zero or below about 100 picoseconds)

it might be worth running just this benchmark again, to check that the low incremental execution times are accurate.

Reply:
I tried this on my local x86_64 Linux machine, which is more powerful than the reference machine, and I got 64/64/65 for the incremental CPU usage for the 3 weights in this file.

The minimum execution times were very similar to those in this PR.